docs/docs/cmd/purview/retentioneventtype/retentioneventtype-list.md

```diff
@@ -0,0 +1,72 @@
+# purview retentioneventtype list
+
+Get a list of retention event types
+
+## Usage
+
+```sh
+m365 purview retentioneventtype list [options]
+```
+
+## Options
+
+--8<-- "docs/cmd/_global.md"
+
+## Examples
+
+Get a list of retention event types
+
+```sh
+m365 purview retentioneventtype list
+```
+
+## Remarks
+
+!!! attention
+    This command is based on a Microsoft Graph API that is currently in preview and is subject to change once the API reached general availability.
+!!! attention
```

Comment:
This part should be included as a More information section, instead of as a note. See https://pnp.github.io/cli-microsoft365/cmd/purview/retentioneventtype/retentioneventtype-get/